shared/packages/worker/src/worker/lib/lib.ts

```diff
@@ -69,3 +69,5 @@ export interface AccessorWithPackageContainer<T extends PackageContainerOnPackag
 	accessorId: string
 	prio: number
 }
+
+export const MAX_EXEC_BUFFER = 10_486_750
```

Comment:
This could use a comment describing why the value is what it is

Reply:
Good point!